docs/guides/qiskit-addons-sqd.mdx

```diff
@@ -0,0 +1,75 @@
+
+### Install from PyPI
+
+The most straightforward way to install the `qiskit-addon-sqd` package is via PyPI.
+
+```bash
+pip install qiskit-addon-sqd
+```
+
+### Build from source
+
+<details>
+<summary>
+Click here to read how to install this package manually
+
+</summary>
+If you wish to contribute to this package or want to install it manually, first clone the repository:
+
+```bash
+git clone git@github.com:Qiskit/qiskit-addon-sqd.git
+```
+
+and install the package via `pip`. The repository also contains example notebooks that you can run. If you plan on developing in the repository, you can install the `dev` dependencies.
+
+Adjust the options to suit your needs.
+
+```bash
+pip install tox notebook -e '.[notebook-dependencies, dev]'
+```
+</details>
+
+
+## Theoretical background
+
+The SQD workflow using self-consistent configuration recovery is explained in detail in [[1]](#references). This section provides an overview to the technique depicted in the following diagram.
+
+![SQD diagram depicting configuration recovery, collecting subsamples, and obtaining eigenstates from those subsamples](/images/guides/qiskit-addons/sqd_diagram.png)
+
+Here $\bar{\mathcal{X}}$ is a set of noisy samples which contain, in the context of the Hamiltonian being simulated, physical and non-physical configurations (represented as bitstrings) obtained from execution on a QPU. The non-physical configurations are due to noise and can be processed by the `sqd.configuration_recovery.recover_configurations()` method to refine the samples into a new set $\mathcal{X}_R$. 
+
+From this set, batches of configurations $\mathcal{S}^{(1)}...\ \mathcal{S}^{(K)}$ are collected and represent samples obtained from different subspaces of the Hamiltonian. These subspace samples are built from a set of basis states, $\mathcal{S}^{(k)}=\{|\mathbf{x}^{(k)}\rangle\}$, and are projections of the system Hamiltonian, $\hat{H}$. This projection is defined as
+
+$$ \hat{H}_{S^{(k)}} = \hat{P}_{\mathcal{S}^{(k)}}\hat{H}\hat{P}_{\mathcal{S}^{(k)}}\text{, with } \hat{P}_{\mathcal{S}^{(k)}} = \sum_{\mathbf{x} \in \mathcal{S}^{(k)}} |\mathbf{x}\rangle\langle\mathbf{x}|,$$
+
+where $\hat{H}_{\mathcal{S}^{(k)}}$ is the Hamiltonian of a given subspace.
+
+The bulk of the SQD workflow lies here wherein each of these subspace Hamiltonians is diagonalized. The ground states obtained from each of these subspaces, $|\psi^{(k)}\rangle$, are used to obtain an estimate of a reference vector of occupancies $\mathbf{n}^{(K)}$ averaged over each of the $K$ subspaces and sent back to the configuration recovery step. A new set of subspaces are then obtained and diagonalized, and this procedure iterates in a self-consistent loop until the ground state energy estimate has converged.
+
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
```

Comment:
I'm not exactly sure how this will render (is it just some tiny link at the bottom of the page?), but these next steps don't seem to be particularly natural next steps

Reply:
You can check out a preview of the docs at https://qiskit.github.io/documentation/pr-2148/. Every PR from someone with write access has a link preview like this - scroll up in the PR history to see it

<img width="1023" alt="Screenshot 2024-11-07 at 2 01 19 PM" src="https://github.com/user-attachments/assets/594dad57-cef7-4b3b-a348-7f1488c770d5">

Reply:
Yeah thanks for pointing that out. I think I had copied it from a different page b/c we usually want to have a "Next Steps" box, but hadn't updated it to be relevant to SQD